circle.yml

```diff
@@ -53,6 +53,7 @@ dependencies:
 
 test:
   override:
+    - make test-doc
```

Comment:
It does happen on circle https://circleci.com/gh/mne-tools/mne-python/462.
It fails because ``cited_mne`` has dependency on ``requests`` (among others), which the circle build doesn't have.